frontend/src/modules/entitieslist/components/EntitiesList.js

```diff
@@ -107,12 +107,22 @@ export class EntitiesListBase extends React.Component<InternalProps> {
 
         // Scroll to selected entity when entity changes
         // and when entity list loads for the first time
+
         if (
             previous.entity !== current.entity ||
             (!prevProps.entities.entities.length &&
                 this.props.entities.entities.length)
         ) {
-            this.scrollToSelectedElement();
+            const list = this.list.current;
+            const element = list.querySelector('li.selected');
+
+            if (!element) {
+                return;
+            } else
```

Comment:
You can remove that `else` and dedent the `element.scrollIntoView()` block underneath.
(You might add a blank line after the `if` block though.)

After the `if` with a `return` statement, there's no need for the `else` part.

```js
if (condition) {
  return something;
} else {
  // do another thing
}
```

is logically the same as

```js
if (condition) {
  return something;
}

// do another thing
```

because `// do another thing` only executes when the condition is `false` anyways (because you `return` early if condition is `true`).

Reply:
I got it. I'm going to correct it right away